easybuild/easyconfigs/t/Trycycler/Trycycler-0.5.3-foss-2021b.eb

```
@@ -0,0 +1,41 @@
+easyblock = 'PythonPackage'
+
+name = 'Trycycler'
+version = '0.5.3'
+
+homepage = 'https://github.com/rrwick/Trycycler'
+description = "Trycycler is a tool for generating consensus long-read assemblies for bacterial genomes."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://github.com/rrwick/Trycycler/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['9a3a512284a91c8a221e3f4dedbc8acabd8f7b93937ad46402d93eee6c7ee95b']
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('R', '4.1.2'),
+    ('edlib', '1.3.9'),
+    ('miniasm', '0.3-20191007'),
+    ('minimap2', '2.24'),
```

drop to 2.22 to match other easyconfigs in this toolchain generation

```suggestion
    ('minimap2', '2.22'),
```

---

thanks for the fix!